official/resnet/resnet_run_loop.py

```diff
@@ -211,9 +214,17 @@ def resnet_model_fn(features, labels, mode, model_class,
   # Generate a summary node for the images
   tf.summary.image('images', features, max_outputs=6)
 
-  model = model_class(resnet_size, data_format, version=version)
+  if dtype == tf.float16:
+    features = tf.cast(features, tf.float16)
+
+  model = model_class(resnet_size, data_format, version=version, dtype=dtype)
+
   logits = model(features, mode == tf.estimator.ModeKeys.TRAIN)
 
+  # This acts as a no-op if the logits are already in fp32. If dtype is
+  # tf.float16, logits must be casted to fp32 for numerical stability.
```

Comment:
nit: must be cast to...

Reply:
Fixed. (Turns out casted is the old English version)